build.json

```diff
@@ -38,7 +38,8 @@
             "configuration/roles.json": "roles.d/appkernels.json",
             "configuration/setup.json": "setup.d/appkernels.json",
             "configuration/rest.d": "rest.d",
-            "configuration/assets.d": "assets.d"
+            "configuration/assets.d": "assets.d",
+            "configuration/modules.d/appkernels.json": "modules.d/appkernels.json"
```

Comment:
If I'm reading this correctly, this specifies that `configuration/modules.d/appkernels.json` is mapped to `etc/modules.d/appkernels.json`, but I don't see `configuration/modules.d/appkernels.json` in this pull request or the 7.0 or 7.1 branches of xdmod-appkernels.